fapolicy_analyzer/tests/test_xdg_utils.py

```diff
@@ -16,105 +16,63 @@
 import os
 
 from fapolicy_analyzer.util.xdg_utils import (
-    xdg_config_dir_prefix,
-    xdg_data_dir_prefix,
-    xdg_state_dir_prefix,
+    app_config_dir_prefix,
```

Comment:
There are now binding functions that provide the directory path as a string. The paths reflect whether or not XDG was enabled in the build.

- app_log_dir
- app_data_dir
- app_config_dir